samples/templates/default-azuredeploy.json

```diff
@@ -7,7 +7,16 @@
             "minLength": 3,
             "maxLength": 24,
             "metadata": {
-                "description": "Name of the DICOM service Web App."
+                "description": "Name of the DICOM Service Web App."
+            }
+        },
+        "functionAppName": {
```

Comment:
Is there a opportunity to auto-generate this name, since this is mostly in background?

Reply:
Yeah, we'll generate this automatically if left unspecified